pkg/querier/distributor_queryable.go

```diff
@@ -129,15 +132,28 @@ func (q *distributorQuerier) streamingSelect(ctx context.Context, minT, maxT int
 		sets = append(sets, newTimeSeriesSeriesSet(results.Timeseries))
 	}
 
+	traceID, _ := tracing.ExtractTraceID(ctx)
+	var chunkInfo *chunkinfologger.ChunkInfoLogger
+	if chunkinfologger.IsChunkInfoLoggingEnabled(ctx) {
+		chunkInfo = chunkinfologger.NewChunkInfoLogger("ingester message", traceID, q.logger, chunkinfologger.ChunkInfoLoggingFromContext(ctx))
+		q.logger.Log("msg", "ingester streamSelect: start", "traceid", traceID, "mint", minT, "maxt", maxT)
```

Comment:
Should we move this into `ChunkInfoLogger` and log similar information for store-gateway requests too?

Reply:
added